### PR TITLE
add dataflow bucket info

### PR DESCRIPTION
**Pull request by iwko**

### CODE REVIEW

File: docs/technologies.md [hunk 1]
```diff
@@ -71,6 +71,13 @@ separate things. Your Beam process might need just a subset of the project requi
 options.view_as(SetupOptions).requirements_file = resolve(get_resource_absolute_path('my-beam-process-requirements.txt', Path(__file__)))
 ```
 
+Note that in those lines:
```
Comment (beczkowb) [inline docs/technologies.md:74]: Repetition from the previous sentence "Note that the project requirements".

Comment (beczkowb) [inline docs/technologies.md:74]: Say what you want to say first, and then, provide a code sample (not in the middle of the sentence).

File: docs/technologies.md [hunk 2]
```diff
@@ -71,6 +71,13 @@ separate things. Your Beam process might need just a subset of the project requi
...
+```python
+google_cloud_options.staging_location = f"gs://{staging_location}"
+google_cloud_options.temp_location = f"gs://{temp_location}"
+```
+we use a Google Cloud Storage Bucket links. This bucket is not created automatically and you need to create it by yourself with separate directories for staging and temp.
```
Comment (beczkowb) [inline docs/technologies.md:79]: Beam uses Cloud Storage Bucket, not us.

Comment (beczkowb) [inline docs/technologies.md:79]: BigFlow doesn't create these buckets automatically.

Comment (iwko) [reply docs/technologies.md:79]: By automatically i mean bigflow/composer

**Pull request merged**